assets/js/util/chart.js

```diff
@@ -73,3 +73,25 @@ export const isSingleSlice = ( report ) => {
 
 	return false;
 };
+
+/**
+ * Calculates difference between two values.
+ *
+ * @since n.e.x.t
+ *
+ * @param {number} currentValue  The difference which can be positive or negative.
+ * @param {number} previousValue Options to be passed to the ChangeArrow component.
+ * @return {number} The difference.
+ */
+export const calculateDifference = ( currentValue, previousValue ) => {
```

Comment:
Let's probably rename this function to be more specific. Something like `calculateDifferenceBetweenChartValues` will better explain what the function does. What do you think?

Reply:
Yes, it makes more sense and will communicate the fact that it's specifically for finding difference in chart value and not a general solution. Updated.